addons/website_blog/models/website_blog.py

```diff
@@ -242,6 +242,19 @@ def write(self, vals):
             vals['is_published'] = False
         for post in self:
             copy_vals = dict(vals)
+
+            # TODO maybe to do inside the mixin
+            # prevent updates on list displays from destroying resize_class
+            if 'cover_properties' in copy_vals:
+                cover_properties = json_safe.loads(copy_vals['cover_properties'])
+                resize_class = cover_properties.get('resize_class', '').split()
+                if 'o_half_screen_height' not in resize_class and \
+                        'o_full_screen_height' not in resize_class and \
+                        'cover_auto' not in resize_class:
+                    old_cover_properties = json_safe.loads(post.cover_properties)
+                    cover_properties['resize_class'] = old_cover_properties.get('resize_class', 'o_half_screen_height')
+                    copy_vals['cover_properties'] = json_safe.dumps(cover_properties)
```

Comment:
- Is that a fix?
- Probably to be done in the mixin indeed

Reply:
I could produce the problem in 14.0 by changing the background color from the blogs page.
(The background color could not be changed in 13.0 - but the size is reset if the cover image is changed which seems more like a feature.)
I will extract this into the mixin in a 14.0 PR.

Reply:
Extracted to https://github.com/odoo/odoo/pull/77018 in 14.0